source/query/queries.txt

```diff
@@ -1,25 +1,59 @@
-.. _past-query-view:
-
-=================
-View Past Queries
-=================
+=======================================
+Managing Saved Queries and Aggregations
+=======================================
 
 .. default-domain:: mongodb
 
-Compass automatically stores up to 20 most recent queries for each
-collection. To view these queries, click the history icon in the query bar.
+Saving an Aggregation Pipeline
```

Comment:
It seems a bit weird to jump right from the title to a header. Can we add some sort of brief intro like:

Starting in Compass <version>, you can load saved queries and aggregations from the :guilabel:`My Queries` view once you are connected to your cluster. This page explains how to add and view favorite queries and aggregations."